lib/PayPal/Api/Patch.php

```diff
@@ -7,13 +7,13 @@
 /**
  * Class Patch
  *
- * A JSON patch object used for applying partial updates to resources.
+ * A JSON Patch object used for doing partial updates to resources.
  *
  * @package PayPal\Api
  *
  * @property string op
  * @property string path
- * @property mixed  value
+ * @property string value
```

Comment:
This will actually be a `mixed` type. It basically means, it could be anything more or less. The value could be a `string` or an `integer` based on what you are patching.